packages/synapse-react-client/src/components/SynapseTable/TopLevelControls/TopLevelControls.tsx

```diff
@@ -174,6 +199,29 @@ const TopLevelControls = (props: TopLevelControlsProps) => {
           )}
         </div>
         <div className="TopLevelControls__actions">
+          {topLevelCustomControls &&
+            topLevelCustomControls.map(customControl => {
+              return (
+                <>
+                  <Button
+                    variant="text"
+                    disabled={!numberOfResultsToInvokeAction}
+                    onClick={() =>
+                      customControl.onClick({
+                        data,
+                        selectedRows,
+                        refresh,
+                        request: getLastQueryRequest(),
+                      })
+                    }
+                    startIcon={customControl.icon}
+                  >
+                    {customControl.buttonText}
+                  </Button>
+                  <Divider orientation="vertical" variant="middle" flexItem />
+                </>
+              )
+            })}
```

Comment:
Nit, need a `key` for mapping jsx
```suggestion
            topLevelCustomControls.map((customControl, index) => {
              return (
                <React.Fragment key={index}>
                  <Button
                    variant="text"
                    disabled={!numberOfResultsToInvokeAction}
                    onClick={() =>
                      customControl.onClick({
                        data,
                        selectedRows,
                        refresh,
                        request: getLastQueryRequest(),
                      })
                    }
                    startIcon={customControl.icon}
                  >
                    {customControl.buttonText}
                  </Button>
                  <Divider orientation="vertical" variant="middle" flexItem />
                </React.Fragment>
              )
            })}
```

Reply:
Added key